stdlib/public/core/Optional.swift

```diff
@@ -418,12 +418,18 @@ extension Optional where Wrapped: ~Copyable {
   ///
   /// - Returns: The wrapped value being stored in this instance. If this
   ///   instance is `nil`, returns `nil`.
-  @_alwaysEmitIntoClient // FIXME(NCG): Make this public.
-  public mutating func _take() -> Self {
+  @_alwaysEmitIntoClient
+  public mutating func take() -> Self {
     let result = consume self
     self = nil
     return result
   }
+
+  @available(*, deprecated, renamed: "take")
```

Comment:
This was never public, I don't see why we need to deprecate it.

Reply:
This is a conservative move, to avoid breaking any code (outside of this repo) that may have already adopted the `_take` spelling since it landed earlier this year.

Reply:
Removed! There is little point to keeping it on this branch, and the risk is marginal even on release/6.0